lib/inputs/dovetail-impressions.js

```diff
@@ -85,6 +84,25 @@ module.exports = class DovetailImpressions {
         cause:            info.cause
       }
     };
+
+    if (imp.vast) {
+      out.json.vast_advertiser = imp.vast.advertiser;
+
+      if (imp.vast.ad) {
+        out.json.vast_ad_id = imp.vast.ad.id;
+      }
+
+      if (imp.vast.creative) {
+        out.json.vast_creative_id = imp.vast.creative.id;
+      }
+
+      if (imp.vast.pricing) {
+        out.json.vast_price_value = parseFloat(imp.vast.pricing.value);
```

Comment:
not totally positive I need to parse this, but jic

Reply:
In theory, JSON supports floats.  But yeah, you never know!

Reply:
I am semi-concerned that the ruby gem updating BQ need this to be in the correct data type for the insert to work